src/app/community-help/render.tsx

```diff
@@ -21,10 +21,18 @@ import { ArchiveSearchBar } from './ArchiveSearchBar'
 import classes from './index.module.scss'
 
 export const CommunityHelp: React.FC = () => {
+  const [searchState, setSearchState] = useState('loading')
+
   const { hits }: { hits: Array<any> } = useHits()
 
   const hasResults = hits && Array.isArray(hits) && hits.length > 0
```

Comment:
if hits.length === 0, that would mean that this is no longer loading. It just has zero results. In which case, you'd want to show the no results screen. Logic needs a revisit here